lochness/redcap/__init__.py

```diff
@@ -80,17 +80,19 @@ def initialize_metadata(Lochness: 'Lochness object',
         Lochness, study_name, f'redcap.{project_name}'))
 
     # sources to add to the metadata, apart from REDCap, XNAT, and Box
-    source_source_name_dict = {'mindlamp': 'Mindlamp'}  
+    source_source_name_dict = {'mindlamp': ['Mindlamp', 'chrdig_lamp_id']}
```

Comment:
This consists of only one element. Why do we have a for loop below then?

Reply:
Keeping it as a loop would make future dev easier when including additional data sources with different ID mappings.